PlayDate/Views/Cells/MatchPicturesTableViewCell.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  MatchPicturesTableViewCell.swift
+//  PlayDate
+//
+//  Created by Michael Cubero on 12/4/21.
+//
+
+import UIKit
+
+class MatchPicturesTableViewCell: UITableViewCell {
```

Comment:
So for all of the match pictures, we might have to do a horizontal swiping collection view instead, but that might be out of scope rn since the pictures still do appear. If you have time, you could probably reference [Spotify Browse Collection View](https://www.youtube.com/watch?v=jDltg1nRAcY&list=PL5PR3UyfTWve9ZC7Yws0x6EGjBO2FGr0o&index=10) or [Categories Collection View](https://www.youtube.com/watch?v=Zojf15WK8mc&list=PL5PR3UyfTWve9ZC7Yws0x6EGjBO2FGr0o&index=16)

We'll need this area to be dynamic since we don't know how many photos will be available